lib/client/state.es6.js

```diff
@@ -79,18 +79,36 @@ foam.CLASS({
       of: 'org.chromium.apis.web.CompatProperty',
       documentation: 'Compat data properties associated with releases in view.',
       name: 'selected',
+      preSet: function(old, nu) {
+        if (!this.selectable) return nu;
+
+        const indexOf = (prop) => {
```

Comment:
const indexOf = prop => this.selectable.findIndex(sProp => sProp.name === prop.name)